build/chrome.go

```diff
@@ -40,7 +40,11 @@ func (c *Chrome) Build() error {
 
 	if pkgSrcPath != "" {
 		srcDir = "chrome/local"
-		pkgDestPath := filepath.Join(devDestDir, "google-chrome.deb")
+		err := os.MkdirAll(devDestDir+"/"+srcDir, 0755)
```

Comment:
Should use `filepath.Join` to make file path separator platform-independent. Just declare `pkgDestDir := filepath.Join(devDestDir, srcDir)` and also use this value in `pkgDestPath` initialization.